core/vm/environment.libevm.go

```diff
@@ -118,13 +132,17 @@ func (e *environment) callContract(typ CallType, addr common.Address, input []by
 			}
 		case nil:
 		default:
-			return nil, gas, fmt.Errorf("unsupported option %T", o)
+			return nil, fmt.Errorf("unsupported option %T", o)
 		}
 	}
 
 	if in.readOnly && value != nil && !value.IsZero() {
-		return nil, gas, ErrWriteProtection
+		return nil, ErrWriteProtection
 	}
+	if !e.UseGas(gas) {
+		return nil, ErrOutOfGas
```

Comment:
nit same here, wrapping the error

```suggestion
		return nil, fmt.Errorf("%w", ErrOutOfGas)
```

Reply:
As with https://github.com/ava-labs/libevm/pull/73#discussion_r1888899560